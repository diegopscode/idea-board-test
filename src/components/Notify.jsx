import React from 'react'

const Notify = ({ show }) => (
    show &&
	<div className="notify">
		<div className="notify-alert">All changed saved!</div>
	</div>
)

export default Notify
