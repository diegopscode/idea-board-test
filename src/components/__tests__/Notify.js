import React from 'react'
import renderer from 'react-test-renderer'

import Notify from '../Notify'

describe("NotifyComponent", () => {
    test('snapshot renders notify component', () => {
        const component = renderer.create(<Notify />)
        let notify = component.toJSON()
        expect(notify).toMatchSnapshot()
    })
})