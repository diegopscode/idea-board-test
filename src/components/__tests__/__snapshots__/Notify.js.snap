// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotifyComponent snapshot renders notify component 1`] = `
<div
  className="notify"
>
  <div
    className="notify-alert"
  >
    All changed saved!
  </div>
</div>
`;
